molgenis-data/src/main/java/org/molgenis/data/event/BootstrappingEventPublisher.java

```diff
@@ -0,0 +1,22 @@
+package org.molgenis.data.event;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BootstrappingEventPublisher
+{
+	private final ApplicationEventPublisher publisher;
+
+	@Autowired
+	public BootstrappingEventPublisher(ApplicationEventPublisher publisher)
+	{
+		this.publisher = publisher;
+	}
+
+	public void createBoostrapEvent(boolean done)
```

Comment:
rename to `publishBootstrappingFinishedEvent()` and remove parameter to make intention clearer?